walkthroughs/2-fuse-aggregator-and-api-management/walkthrough.adoc

```diff
@@ -216,28 +216,48 @@ Verify that you followed each step in the procedure above.  If you are still hav
 ----
 
 . Leave the *Description* field empty.
-
-. Click *Add API* at the bottom of the screen.
-
-. From the *Overview* screen, select the *Configure APIcast* button.
 // The 'fuse-aggregation-app-url' should be the url of the Fuse Aggregation App e.g. https://fuse-flights-aggregator-ak49.cluster-lfa3xlh.opentry.me/
-. In the *Private Base URL* field, enter:
+. In the *Private endpoint* field, enter:
 +
 [subs="attributes+"]
 ----
 {route-fuse-flights-aggregator-host}
 ----
+. Click *Create Backend* at the bottom of the screen.
+
+=== Adding a new Product to Red Hat 3scale
+
+. Select *Dashboard* from the top navigation menu.
+. In the *API* section of the Dashboard, select the *Products* tab.
+. Click *New Product*.
+. Enter the following as the *Name* and *System name*:
++
+[subs="attributes+"]
+----
+{fuse-flights-aggregator-app-name}
+----
+. Leave the *Description* field empty.
+. Click *Create Product* at the bottom of the screen.
```

Comment:
don't we need a verification here?

Reply:
We can add one asking do they now see a the newly created product in the *APIs* section if that works for you?